components/create-collective/sections/CreateCollectiveForm.js

```diff
@@ -0,0 +1,292 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Formik, Field, Form } from 'formik';
+import { Flex, Box } from '@rebass/grid';
+import { assign, get } from 'lodash';
+import { FormattedMessage, defineMessages, injectIntl } from 'react-intl';
+
+import { H1, P } from '../../Text';
+import Container from '../../Container';
+import Illustration from '../../home/HomeIllustration';
+import StyledCheckbox from '../../StyledCheckbox';
+import StyledInput from '../../StyledInput';
+import StyledInputField from '../../StyledInputField';
+import StyledInputGroup from '../../StyledInputGroup';
+import StyledButton from '../../StyledButton';
+import MessageBox from '../../MessageBox';
+import Link from '../../Link';
+
+class CreateCollectiveForm extends React.Component {
+  static propTypes = {
+    error: PropTypes.string,
+    host: PropTypes.object,
+    query: PropTypes.object,
+    collective: PropTypes.object,
+    loading: PropTypes.bool,
+    onSubmit: PropTypes.func,
+    intl: PropTypes.object.isRequired,
+    onChange: PropTypes.func,
+  };
+
+  constructor(props) {
+    super(props);
+    this.handleChange = this.handleChange.bind(this);
+
+    const collective = { ...props.collective }; // {}
+
+    this.state = {
+      collective,
+      tosChecked: false,
+      hostTosChceked: false,
+    };
+
+    this.messages = defineMessages({
+      introduceSubtitle: {
+        id: 'createCollective.subtitle.introduce',
+        defaultMessage: 'Introduce your Collective to the community.',
+      },
+      back: {
+        id: 'createCollective.link.back',
+        defaultMessage: 'Back',
+      },
+      header: { id: 'createCollective.header.create', defaultMessage: 'Create a Collective' },
+      nameLabel: { id: 'createCollective.form.nameLabel', defaultMessage: "What's the name of your collective?" },
+      slugLabel: { id: 'createCollective.form.slugLabel', defaultMessage: 'What URL would you like?' },
+      descLabel: { id: 'createCollective.form.descLabel', defaultMessage: 'What does your collective do?' },
+      createButton: {
+        id: 'createCollective.button.create',
+        defaultMessage: 'Create Collective',
+      },
+    });
+  }
+
+  handleChange(fieldname, value) {
+    const collective = {};
+
+    collective[fieldname] = value;
+
+    this.setState({
+      collective: assign(this.state.collective, collective),
+    });
+  }
+
+  render() {
+    const { intl, error, query, host } = this.props;
+
+    console.log(host);
+
+    const initialValues = {
+      name: '',
+      desc: '',
```

Comment:
Can we name that `description` ?

Reply:
Sorry, I definitely changed this before, not sure why it's gone back